data/User/createUser.js

```diff
@@ -128,9 +128,18 @@ async function verifyAndCleanCreateUserParams(
  * @param {String} rawPwd The user's password as plaintext
  * @param {String[] | String} accessGroups The access groups the user belongs to
  * @param {String} role The user's role in the system
+ * @param {Boolean} isCreateeAdmin Whether the access groups should be granted or simply applied for
  * @return {Object}
  */
-async function createUser(email, fName, lName, rawPwd, accessGroups, role) {
+async function createUser(
+    email,
+    fName,
+    lName,
+    rawPwd,
```

Comment:
Make this a more semantically meaningful name!

Reply:
Also from @vrabelt : We should implement using creator ObjectId rather than a boolean, for security and cleaner routes.